### PR TITLE
Gidk 977 dev

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss [hunk 1]
```diff
@@ -56,9 +53,44 @@
...
+    justify-content: flex-end;
+    a{
+        display: flex;
+        width: 32px;
+        height: 32px;
+        justify-content: center;
+        align-items: center;
+        background-color: #fff;
+        font-size: 16px;
+        color: #1a237e !important;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss:68]: Use color variable in color and background color

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss [hunk 2]
```diff
@@ -56,9 +53,44 @@
...
+        border: 1px solid $gainsboro;
+    }
+}
+.d-inline-flex{
+    display: inline-flex;
+}
+.glyphicon-triangle-bottom{
+z-index: 1;position: absolute; left: 152px;font-size: 9px;color: $very-light-gray;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss:76]: Format the styles (format only this section and not whole file)

File: apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
-}
+@import "../../../../../assets/css/color.scss";
 .liabilities-assets {
     max-width: 760px;
     width: 100%;
 }
+.trial-balance{
+    span{
+        i.glyphicon-search{
+        position: relative;z-index: 1; font-size: 13px;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss:9]: Acid writing styles in single line

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    .pl-table-grid {
+        th{
+            border: none !important;
+            border-bottom:1px solid $gainsboro !important;
+            padding: 13px 8px 14px 0 !important;
+            &:first-child{
+            padding: 0 !important;
+            }
+                i.glyphicon-search{
+                position: relative;z-index: 1; font-size: 13px;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss:43]: Avoid writing styles in single line

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.html [hunk 5]
```diff
@@ -186,7 +185,7 @@ <h1>Sorry, No Data Found!</h1>
                 </div>
 
                 <div class="account clearfix" *ngIf="plData && plData?.incomeStatment"
-                    [ngStyle]="{'border-bottom':plData?.incomeStatment.incomeBeforeTaxes.type === 'DEBIT' ? '1px solid #dbdde3' : 'none' }">
+                    [ngStyle]="{'border-bottom':plData?.incomeStatment.incomeBeforeTaxes.type === 'DEBIT' ? 'none' : 'none' }">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.html:188]: NgStyle is no longer needed as condition failure or success are doing the same thing

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss [hunk 6]
```diff
@@ -0,0 +1,64 @@
+@import "../../../../../assets/css/color.scss";
+   :host ::ng-deep .table-container {
+      padding: 0;
+    }
+
+    :host ::ng-deep .table-container .profitLoss section div > div {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss:6]: Avoid camelCase in class names

File: apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+.bs-table-grid {
+    th{
+        border: none !important;
+        border-bottom:1px solid $gainsboro !important;
+        padding: 13px 8px 14px 0 !important;
+        &:first-child{
+            padding: 0 !important;
+        }
+            i.glyphicon-search{
+            position: relative;z-index: 1; font-size: 13px;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss:11]: Avoid writing styles in single line

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss:6]: profitLoss class and text is appearing many palces (28 files) including TS file , so I am avoiding this class name

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss:68]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.scss]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.scss]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/pl/pl-grid/pl-grid.component.html]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/bs/bs-grid/bs-grid.component.scss]: Done

**Pull request merged**